packages/@jsii/kernel/test/kernel.test.ts

```diff
@@ -95,7 +95,7 @@ defineTest.skip = function (
       });
       expect(result.result).toBe(0o644);
 
-      return closeRecording(kernel);
+      return await closeRecording(kernel);
```

Comment:
There was a bug right there that this change allows to prevent! Thankfully, this bug was in a test so it did not get to cause actual harm.